tests/Functional/Doctrine/Orm/RedirectRouteTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\RoutingBundle\Tests\Functional\Doctrine\Orm;
+
+use Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm\Route;
+use Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm\RedirectRoute;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\RouteCollection;
+
+class RedirectRouteTest extends OrmTestCase
+{
+    private $repository;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->clearDb(Route::class);
+
+        $this->repository = $this->getContainer()->get('cmf_routing.route_provider');
+    }
+
+    public function testRedirectDoctrine()
+    {
+        $route = $this->createRoute('route1', '/test');
+
+        $redirectRouteDoc = new RedirectRoute();
+        $redirectRouteDoc->setRouteName('redirect1');
+        $redirectRouteDoc->setRouteTarget($route);
+        $redirectRouteDoc->setPermanent(true);
+
+        $this->getDm()->flush();
```

Comment:
is this flush needed? you flush again below.

and are you sure you do not need to persist the $redirectRouteDoc?